var/spack/repos/builtin/packages/itk/package.py

```diff
@@ -58,13 +58,18 @@ class Itk(CMakePackage):
     depends_on("expat")
     depends_on("fftw-api")
     depends_on("googletest")
-    depends_on("hdf5+cxx")
+    depends_on("hdf5+cxx+hl~mpi")
```

Comment:
Is `~mpi` required for the itk to build? If not it might be better to omit it for now as this will restrict anyone from installing it in a stack with an hdf5 built with mpi support.

Reply:
No, good catch, it's not required. Fixed.